spec/controllers/api/v1/book_suggestion_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe Api::V1::BookSuggestionsController do
+  describe 'POST #create' do
+    context 'When creating a valid book suggestion' do
+      let!(:book_suggestion_attributes) { attributes_for(:book_suggestion) }
+      let(:request) { post :create, params: { book_suggestion: book_suggestion_attributes } }
+
+      it 'creates a new book suggestion' do
+        expect { request }.to change { BookSuggestion.count }.by(1)
+      end
+
+      it 'responds with 201 status' do
+        post :create, params: { book_suggestion: book_suggestion_attributes }
+        expect(response).to have_http_status(:created)
+      end
+    end
+
+    context 'When creating an invalid book suggestion' do
+      let!(:book_suggestion_attributes) { attributes_for(:book_suggestion, title: nil) }
+      let(:request) { post :create, params: { book_suggestion: book_suggestion_attributes } }
+
+      before do
+        post :create, params: { book_suggestion: book_suggestion_attributes }
```

Comment:
Also here, you can call the `request` of the line 21.
By the way, if you put the let of the line 21 next to `describe POST #create`, and then in each context you change the `book_suggestion_attributes` value, you can call `request` and it will always work.